.github/workflows/ci.yml

```
@@ -23,6 +23,18 @@ jobs:
+        run: npm test
+
+      - name: Set up Java 17 for SonarScanner
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: 17
+
+      - name: SonarCloud Scan
+        uses: SonarSource/sonarqube-scan-action@v1
+        env:
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)